test/test_create_floor_array.c

```diff
@@ -0,0 +1,62 @@
+/**
+ * @file test_create_floor_array.c
+ *
+ * @brief To test create_floor_array function which creates array for every floor.
+ *
+ *
+ * @author Hariprasad Munusamy  - HariprasadMunusamy@cmail.carleton.ca
+ * @return Writes the test case log in output file
+ */
+#include<stdio.h>
+#include<stdlib.h>
+#include<stdbool.h>
+//#include "functions.h"
+
+struct passenger{
+bool	in_elevator; //!< Stores whether passegner is in elevator. 
+unsigned int	arrival_time; //!< stores arrival time of the passenger
+unsigned int	arrival_floor; //!< stores the arrival_floor of the passenger
+unsigned int	dest_floor; //!< stores the destination floor of the passenger
+unsigned int 	id; //!< stores the passenger_id of the passenger
+struct passenger*next; //! stores address of the next node in case of creation of queues
+};
+
+struct passenger ** create_floor_array(int no_of_floors)
```

Comment:
Make these functions static. so they wont interfere with the other functions we have included